py3status/modules/window_title.py

```diff
@@ -51,7 +51,12 @@ def __init__(self):
     def window_title(self):
         window = find_focused(i3.get_tree())
 
+        if not window or window.get('name') is None or \
+           str(window.get('num')) == window.get('name'):
```

Comment:
hmm, we like to keep modules working the same for existing users.

Now maybe this is actually a bug and so needs fixing (I'm assuming this doesn't happen in `window_title_async`)

If it is not a bug then we should probably add an option to control it.